0001-typedb/01-start/03-quickstart.md

```diff
@@ -0,0 +1,673 @@
+---
+pageTitle: Quickstart guide
+keywords: getting started, typedb, typeql, tutorial, quickstart, overview
+longTailKeywords: get started with typedb, typedb tutorial, typedb quickstart, learn typedb
+summary: Learn how to create a TypeDB database, load schema and data, perform queries.
+toc: false
+---
+
+# Quickstart guide
+
+## Overview
+
+This Quickstart guide goes through the step-by-step process of creating and interacting with a TypeDB database designed 
+for an [Identity and Access Management](https://en.wikipedia.org/wiki/Identity_management) solution that will be 
+described [later](04-iam-schema.md).
+
+<div class="note">
+[Important]
+We highly recommend completing this guide. Its goal is to prepare an environment for TypeDB exploration and development.
+The resulting database (schema and data) will be needed for example queries throughout the documentation.
+</div>
+
+## Prerequisites
+
+This Quickstart guide takes advantage of TypeDB Studio, a desktop GUI tool for interacting with TypeDB. To proceed, 
+please install both TypeDB and TypeDB Studio:
+
+- [Install TypeDB](02-installation.md)
+- [Install TypeDB Studio](../../02-clients/01-studio.md#get-typedb-studio)
+
+IMPORTANT:
+<div class="note">
+[Important]
+It’s recommended to use TypeDB and TypeDB Studio of the same version number in order to ensure compatibility. If 
+TypeDB Studio doesn't have a version similar to the TypeDB release — use the closest lower version instead.
+</div>
+
+## Initialize database
+
+### Start TypeDB
+
+Run the following command in a terminal:
+
+<!-- test-ignore -->
+```bash
+typedb server
```

Comment:
Or .bat for Windows users.

Reply:
Didn't you just recently fix the typedb binary to be able to run with just `typedb server` on Win if we set PATH variable right? The version with fix from you runs from `typedb server` (in a separate window, but still).

Reply:
Hm, I thought it required the .bat extension still. Excellent!

Reply:
Let's test it after the release. Just to be sure, OK? 
At this moment, my Win machine refuses to start TypeDB correctly, but it initiates the launch (and shows the ASCII graphics I believe).